src/preferences/dialog/dlgprefeq.cpp

```diff
@@ -105,28 +103,13 @@ void DlgPrefEQ::slotNumDecksChanged(double numDecks) {
 
         QLabel* label = new QLabel(QObject::tr("Deck %1").arg(deckNo), this);
 
-        QString group = PlayerManager::groupForDeck(
-                m_deckEqEffectSelectors.size());
-
-        m_filterWaveformEnableCOs.append(
-                new ControlObject(ConfigKey(group, "filterWaveformEnable")));
-        m_filterWaveformEffectLoaded.append(false);
-
         // Create the drop down list for deck EQs
         QComboBox* eqComboBox = new QComboBox(this);
         m_deckEqEffectSelectors.append(eqComboBox);
         connect(eqComboBox,
                 QOverload<int>::of(&QComboBox::currentIndexChanged),
                 this,
-                &DlgPrefEQ::slotEqEffectChangedOnDeck);
-
-        // Create the drop down list for Quick Effects
-        QComboBox* quickEffectComboBox = new QComboBox(this);
```

Comment:
I disagree. The skins need to be modified. I'm not going to modify any more skins in this PR. We'll do that after merging this.

Reply:
true, in case of shade you're stuck with the previous quick effect.
though I think a minimal effect selector will do for now. just the arrow, like 20*20px, two of them in the main mixer, no big deal

Reply:
@daschuer Is right. 
Only Tango has the selector, in the other skins there's no way to change the quick effects now.
I know I offered my help with that back then, I found no time for that and I'm not going to rush that now, and I can not promise to do that immediatley after merge. probably not before having released 2.3 beta.

unfortunately no one else noticed this regression until now.
so either you add a minimal, probably ugly selector to the skins (without affecting existing layouts) > unlikely
or we'll have to do that another time and restore the quick effect selector preferences for now.

Reply:
I'm not putting work into moving backwards.

Reply:
Just bring back the quick effect selector to the EQ preferences. 

This is where the users are used to look for it.  Now that we are so flexible you can configure a four band EQ, so the combo box still perfectly fits to the EQ preferences as well. 

There is no need to remove the selector there. Removing this box without a replacement is a regression and blocks this PR.